packages/paste-core/components/visual-picker/src/VisualPickerCheckbox.tsx

```diff
@@ -0,0 +1,125 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import {useUID} from '@twilio-paste/uid-library';
+import {Box, safelySpreadBoxProps} from '@twilio-paste/box';
+import {BaseRadioCheckboxControl, BaseRadioCheckboxLabel} from '@twilio-paste/base-radio-checkbox';
+import {HiddenCheckbox, CheckboxIcon} from '@twilio-paste/checkbox';
+import {ScreenReaderOnly} from '@twilio-paste/screen-reader-only';
+import {useMergeRefs} from '@twilio-paste/utils';
+import {type BorderColor} from '@twilio-paste/style-props';
+
+import {VisualPickerCheckboxContext} from './VisualPickerContext';
+import {type VisualPickerCheckboxProps} from './types';
+
+export const VisualPickerCheckbox = React.forwardRef<HTMLInputElement, VisualPickerCheckboxProps>(
+  ({checked, element = 'VISUAL_PICKER_CHECKBOX', children, id, indeterminate, labelText, onChange, ...props}, ref) => {
+    const [isHovering, setIsHovering] = React.useState(false);
+
+    const visualPickerCheckboxGroupContext = React.useContext(VisualPickerCheckboxContext);
+    const helpTextId = useUID();
+    const labelId = useUID();
+    const checkboxId = id ? id : useUID();
+
+    const handleChange = React.useCallback(
+      (event: React.ChangeEvent<HTMLInputElement>): void => {
+        onChange(event);
+      },
+      [onChange]
+    );
+
+    // Prioritizing direct props values over whatever visualPickerCheckboxGroupContext passes down
+    const disabled = props.disabled != null ? props.disabled : visualPickerCheckboxGroupContext.groupIsDisabled;
+    const name = props.name != null ? props.name : visualPickerCheckboxGroupContext.name;
```

Comment:
`name` is required on visual picker checkbox group and visual picker radio group, so if we prioritize the context value here, there will be no need to keep the prop on visual picker checkbox and visual picker radio. should I go forward with removing them?

Reply:
Hmm well people can still use VisualPickerRadio/Checkbox without the group right?

I just meant like `const name = visualPickerCheckboxGroupContext.name || props.name`

Reply:
the API doesn't explicitly restrict people from using the individual without the group but we have no use cases for it and we're not documenting it. But I'll just swap around the name prioritization to use context over props and leave the name prop as is in case we want to allow that in the future